applications/location-api/helm/links-location-api/templates/NOTES.txt

```diff
@@ -0,0 +1,2 @@
+To start a simulation go to: http://{{ (index .Values.env 0).value }}:{{ (index .Values.simulatorService.ports 0).nodePort }}
```

Comment:
Copyright header should be added here too and to all auto-generated files. `.gitignore` is an exception.